backend/schemas/source/excel/templates/federal-awards-expended-template.jsonnet

```diff
@@ -207,17 +222,65 @@ local open_ranges_defns = [
     'Number of Audit Findings',
     'number_of_audit_findings',
   ],
+  [
+    Sheets.open_range {
+      keep_locked: true,
+      formula: '=IF(A{0}<>"", "AWARD-"&TEXT(ROW()-1,"00000"), "")',
+      width: 24,
+      help: Help.unknown,
+    },
+    SV.NoValidation,
+    'Award Index (Read Only)',
+    'award_index',
+  ],
+  [
+    Sheets.open_range {
+      keep_locked: true,
+      formula: '=CONCATENATE(A{0},B{0})',
```

Comment:
For discussion/ticketing for another day:

We should, at this point, create a JSON mini-language for specifying the formulas.

E.g.

```
formula: { 
  function: CONCATENATE
  parameters: [
    { cell: { column: "A", row: CURRENT } },
    { cell: { column: "B", row: CURRENT } },
  ]}
```

or similar. Why? We're already parsing the JSON structure, and then walking it (compiling it) to generate the XLSX docs. Eliminating complex strings (the formulas, for example) and replacing them with a well-defined JSON object, that can be turned into a string with the appropriate values in a clean and reliable way, would make sense.

This is not a comment for this code right now, but a note for us to remember/ticket/consider in the future. Point being: use a data structure as opposed to trying to embed a language/semantics into a string.